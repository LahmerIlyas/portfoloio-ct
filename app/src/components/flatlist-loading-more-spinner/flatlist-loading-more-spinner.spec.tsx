import React from 'react';
import renderer from 'react-test-renderer';
import { FlatListLoadingMoreSpinner } from '.';

describe('<FlatListLoadingMoreSpinner />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<FlatListLoadingMoreSpinner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
