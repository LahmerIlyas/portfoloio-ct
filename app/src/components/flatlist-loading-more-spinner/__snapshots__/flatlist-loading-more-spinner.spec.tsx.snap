// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlatListLoadingMoreSpinner /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": 80,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#91C788"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
